api/db/populate-messages.js

```diff
@@ -1,18 +1,35 @@
 const { getParagraph } = require('./utils');
+const {
+  createNotification,
+  notificationTypes,
+} = require('./populate-notifications');
 
 const Friendship = require('../models/Friendship');
 const Message = require('../models/Message');
 
 let displayLogs;
+let notificationsCount = 0;
 
 const createMessage = async (messageParams) => {
-  const message = new Message({
-    friendship: messageParams.friendship,
-    user: messageParams.user,
-    content: messageParams.content,
-  });
+  const message = new Message(messageParams);
   await message.save();
 
+  const user =
+    message.friendship.receivant.toString() === message.user.toString()
+      ? message.friendship.receivant
+      : message.friendship.requestant;
```

Comment:
```suggestion
      ? message.friendship.requestant
      : message.friendship.receivant;
```
This is what I mentioned earlier that should be the other way around, sorry if the comment was a bit confusing!

Also, I just realized that we are generating messages for all friendships, but only accepted friendships do have chats on the real application, so we need to address this below! A simple check should do :slightly_smiling_face: